cpp/src/slice2swift/Gen.cpp

```diff
@@ -1267,7 +1290,80 @@ Gen::TypesVisitor::visitInterfaceDefEnd(const InterfaceDefPtr&)
 void
 Gen::TypesVisitor::visitOperation(const OperationPtr& op)
 {
-    writeProxyOperation(out, op);
```

Comment:
This is the only place `writeProxyOperation` was called from, so I inlined it.
The code was copy-pasted from `SwiftUtil.cpp` with no changes.